content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -172,7 +172,7 @@ For a reference to old feature gates that are removed, please refer to
 | `PodDeletionCost` | `true` | Beta | 1.22 | |
 | `PodDisruptionConditions` | `false` | Alpha | 1.25 | 1.25 |
 | `PodDisruptionConditions` | `true` | Beta | 1.26 | |
-| `PodHasNetworkCondition` | `false` | Alpha | 1.25 | |
+| `PodReadyToStartContainersCondition` | `false` | Alpha | 1.25 | |
```

Comment:
We retain the old feature gate in the list of removed feature gates. Also, the feature gate `PodReadyToStartContainersCondition` was only added in 1.28 (or will only have been).

Reply:
Just to make sure I understand what you are suggesting:

Add `PodHasNetworkCondition` to removed feature gates and add `PodReadyToStartContainersCondition` to feature-gates.md but have it added for 1.28?

Reply:
Yes.

Reply:
done.